lexicons/tools/ozone/set/querySets.json

```diff
@@ -0,0 +1,51 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.sets.querySets",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Query available sets",
+      "parameters": {
+        "type": "params",
+        "properties": {
+          "limit": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 100,
+            "default": 50
+          },
+          "cursor": {
+            "type": "string"
+          },
+          "namePrefix": {
+            "type": "string"
+          },
+          "sortBy": {
```

Comment:
maybe add a "reverse sort order" flag called `reverseSort` (?) while we are at it? not a blocking need (we could add it later if needed), just a thought.

Reply:
yep, can't recall why I didn't do this in the first place. decided to go with `sortDirection: 'asc' | 'desc'` because we already have this pattern for `queryEvents`